src/ui/CustomLink/CustomLink.tsx

```diff
@@ -14,4 +14,4 @@ function CustomLink({ className, target = '_blank', children, ...restProps }: Re
     );
 }
 
-export default CustomLink;
+export { CustomLink };
```

Comment:
```suggestion
export CustomLink;
```
same for the rest of exports